kernel/h1/src/spi_device.rs

```diff
@@ -16,6 +16,10 @@ use kernel::ReturnCode;
 use spiutils::protocol::flash::AddressMode;
 use spiutils::protocol::flash::OpCode;
 
+fn get_data_byte_or_default(data: &[u8], idx: usize, default: u8) -> u8 {
+    if idx < data.len() { data[idx] } else { default }
```

Comment:
This can be `data.get(idx).unwrap_or(default)`

Reply:
Ha, yeah, I did not know that, thank you! I tried to actually merge the get_data_byte_or_default back into the code, but the resulting line was very hard to read due to references and dereferencing and the resulting need of parantheses. So I just updated the helper function and added a comment.